boards/pine64/default.nix

```diff
@@ -14,6 +14,20 @@
       ./0001-configs-pine64-lts-Enable-SPI-flash.patch
     ];
   };
+  pine64-pineH64A = allwinnerH6 {
+    boardIdentifier = "pine64-pineH64A";
+    defconfig = "pine_h64_defconfig";
+    withPoweroff = false;
```

Comment:
Why `withPoweroff = false;`? (Not that I doubt it could be non-working.)

Reply:
`withPoweroff = false;` seemed to be set for most SBCs in Tow-Boot. I'll enable it and see if the poweroff command works as expected.

Reply:
Hopefully it will work. It works for existing supported Allwinner boards.

Reply:
`withPoweroff = true` gives build errors.

```
builder for '/nix/store/4cs196kifj0dmmddacsp5qy12ynqfmdq-tow-boot-pine_h64-model-b_defconfig-noenv-aarch64-unknown-linux-gnu-2021.10-003-pre.drv' failed with exit code 1; last 10 log lines:
    AR      net/built-in.o
    LDS     u-boot.lds
    LD      u-boot
  /nix/store/hkwgs14jbngmxs5z6v918szwpcs474xb-aarch64-unknown-linux-gnu-binutils-2.35.1/bin/aarch64-unknown-linux-gnu-ld.bfd: lib/efi_loader/efi_runtime.o: in function `efi_reset_system_boot
time':
  /build/u-boot-2021.10/lib/efi_loader/efi_runtime.c:217: undefined reference to `do_poweroff'
  /nix/store/hkwgs14jbngmxs5z6v918szwpcs474xb-aarch64-unknown-linux-gnu-binutils-2.35.1/bin/aarch64-unknown-linux-gnu-ld.bfd: cmd/boot.o:(.u_boot_list_2_cmd_2_poweroff+0x18): undefined refer
ence to `do_poweroff'
  /nix/store/fmngyhzl5rfz23xnc7lrgkc95mqzx8hj-aarch64-unknown-linux-gnu-stage-final-gcc-debug-wrapper-9.3.0/bin/aarch64-unknown-linux-gnu-ld.bfd: line 249: 10688 Segmentation fault      /nix
/store/hkwgs14jbngmxs5z6v918szwpcs474xb-aarch64-unknown-linux-gnu-binutils-2.35.1/bin/aarch64-unknown-linux-gnu-ld.bfd ${extraBefore+"${extraBefore[@]}"} ${params+"${params[@]}"} ${extraAfte
r+"${extraAfter[@]}"}
  make: *** [Makefile:1790: u-boot] Error 139
  make: *** Deleting file 'u-boot'
  builder for '/nix/store/4cs196kifj0dmmddacsp5qy12ynqfmdq-tow-boot-pine_h64-model-b_defconfig-noenv-aarch64-unknown-linux-gnu-2021.10-003-pre.drv' failed with exit code 2
cannot build derivation '/nix/store/jcmx13ck4wk0423a8ancd5y6fy0ql1gl-tow-boot-pine_h64-model-b_defconfig-noenv-aarch64-unknown-linux-gnu-2021.10-003-pre.drv': 1 dependencies couldn't be buil
t
```